.github/workflows/ci.yml

```diff
@@ -67,19 +70,20 @@ jobs:
   otherjdks:
     name: ${{ matrix.script }} on JDK ${{ matrix.java.version }}
     runs-on: ubuntu-latest
-    needs: remainder
+    needs: sanity
     permissions:
       contents: read
     strategy:
       fail-fast: false
       matrix:
         # jspecify-conformance only tested on JDK 21.
-        script: ['cftests-junit', 'cftests-nonjunit', 'cftests-junit-jdk21', 'typecheck-part1', 'typecheck-part2', 'guava', 'plume-lib']
+        script: ['cftests-junit', 'cftests-nonjunit', 'cftests-junit-jdk21',
+          'typecheck-part1', 'typecheck-part2', 'guava', 'plume-lib', 'daikon-part1', 'daikon-part2']
         # JDK 21 used by sanity before
         java: [{version: '8', experimental: false},
           {version: '11', experimental: false},
           {version: '17', experimental: false},
-          {version: '22', experimental: true},
+          {version: '22', experimental: false},
```

Comment:
This change means that the JDK 22 runtime is now also used to run the test cases - before, the JDK 21 runtime was used, as is the case for experimental platforms.
This reveals the failure with the non-junit tests, which is #826. So we either keep JDK 22 experimental or fix #826.

I can't reproduce the failures we get with `:dataflow:busyExpressionTest` and the last few runs don't produce them. Not sure what is causing this.

Reply:
`:dataflow:busyExpressionTest` fails again as soon as I remove the debugging for it in https://github.com/eisop/checker-framework/pull/830/commits/56c9d409a5981dbbaf0e5e4e4b47ec78f101d3db